### PR TITLE
add 'synapses restricted to region'

### LINKED ISSUE #754: relationship between an intrinsic neuron and its neuropil of innervation
Ideally, intrinsic neurons have all of their synapses restricted to a particular region of the CNS. In real life there may be a few stray synapses just outside the region.

Label: 'synapses restricted to region'
Exact synonym: 'intrinsic to'
Definition: 'A relationship between a neuron and a region, where the neuron has essentially all of its synapses in that region.'
Comment: 'Relationship may still be used if a very small number of synapses fall slightly outside the boundaries of the region.'
Parent: RO:0013001 ('has synaptic IO in region')
Domain: CL:0000540 ('neuron') - will be inherited from 'has synaptic terminal in'

I don't think we want to make this a subproperty of input and output relationships, in case there are neurons that do not have one or the other - some sensory neurons might not have synaptic inputs(?)

related to https://github.com/FlyBase/drosophila-anatomy-developmental-ontology/issues/1626

### PR DESCRIPTION
**Pull request by Clare72**
fixes #754

### CODE REVIEW

**Review by gouttegd — DISMISSED** [review]

File: src/ontology/ro-edit.owl [hunk 1]
```diff
@@ -6351,6 +6352,16 @@ AnnotationAssertion(rdfs:comment obo:RO_0013009 "Not restricting range to 'cell'
...
+# Object Property: obo:RO_0013010 (synapses restricted to region)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0013010 "A relationship between a neuron and a region, where the neuron has essentially all of its synapses in that region.")
+AnnotationAssertion(dc:contributor obo:RO_0013010 <https://orcid.org/0000-0002-1373-1705>)
+AnnotationAssertion(dc:date obo:RO_0013010 "2023-08-17T12:17:24Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:RO_0013010 "intrinsic to")
+AnnotationAssertion(rdfs:comment obo:RO_0013010 "Relationship may still be used if a very small number of synapses fall slightly outside the boundaries of the region.")
+AnnotationAssertion(rdfs:label obo:RO_0013010 "synapses restricted to region"@en)
+SubObjectPropertyOf(obo:RO_0013010 obo:RO_0013001)
+
```
Comment (anitacaron) [inline src/ontology/ro-edit.owl:6364]: Please, add an example of usage (annotation IAO:0000112).

**Review by anitacaron — CHANGES_REQUESTED** [review]

**Comment by Clare72** [comment]
also adding property chain over part_of (if neuron has synapses restricted to part of A, its synapses are also restricted to A)

File: src/ontology/ro-edit.owl [hunk 2]
```diff
@@ -6351,6 +6352,17 @@ AnnotationAssertion(rdfs:comment obo:RO_0013009 "Not restricting range to 'cell'
 AnnotationAssertion(rdfs:label obo:RO_0013009 "sends synaptic output to cell"@en)
 SubObjectPropertyOf(obo:RO_0013009 obo:RO_0002120)
 
+# Object Property: obo:RO_0013010 (synapses restricted to region)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0013010 "An optic lobe intrinsic neuron has its synapses restricted to optic lobe neuropil.")
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0013010 "A relationship between a neuron and a region, where the neuron has essentially all of its synapses in that region.")
+AnnotationAssertion(dc:contributor obo:RO_0013010 <https://orcid.org/0000-0002-1373-1705>)
+AnnotationAssertion(dc:date obo:RO_0013010 "2023-08-17T12:17:24Z"^^xsd:dateTime)
```
Comment (anitacaron) [inline src/ontology/ro-edit.owl:6360]: Please use the annotations from `terms: <http://purl.org/dc/terms/>`

**Review by anitacaron — CHANGES_REQUESTED** [review]

**Comment by cmungall** [comment]
Hmm, this is for smuggling in universal quantifications as existentials.

Owl already has a way of saying synapsed to ONLY region X.

We already snuggle with taxon constraints, there are many impracticalities and traps when using universals directly (I go as far as to say it's an anti pattern). For TCs we encode shortcuts. Should we do the same here?

Cc @d

**Comment by cmungall** [comment]
Cc @dosumis

**Comment by cmungall** [comment]
I'm not going to correct my typo above as I quite like the idea of snuggly taxon constraints 🧸

**Comment by addiehl** [comment]
Please incorporate the comments from issue https://github.com/oborel/obo-relations/issues/754.

**Comment by Clare72** [comment]
@addiehl @anitacaron are you happy with this now?

File: src/ontology/ro-edit.owl [hunk 3]
```diff
@@ -6395,6 +6396,17 @@ AnnotationAssertion(rdfs:comment obo:RO_0013009 "Not restricting range to 'cell'
 AnnotationAssertion(rdfs:label obo:RO_0013009 "sends synaptic output to cell"@en)
 SubObjectPropertyOf(obo:RO_0013009 obo:RO_0002120)
 
+# Object Property: obo:RO_0013010 (synapses restricted to region)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0013010 "An optic lobe intrinsic neuron has its synapses restricted to optic lobe neuropil.")
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0013010 "A relationship between a neuron and a region, where all of the functionally relevant input and output synapses are in that region.")
+AnnotationAssertion(terms:creator obo:RO_0013010 <https://orcid.org/0000-0002-1373-1705>)
```
Comment (anitacaron) [inline src/ontology/ro-edit.owl:6403]: Please use `terms:contributor` instead of `terms:creator`. I can make the change here.

**Review by anitacaron — APPROVED** [review]
Technically, I approve. It still needs an ontology review.

**Review by cmungall — APPROVED** [review]

**Pull request merged**